### PR TITLE
allow passing in of enviroment into comb and seq

### PR DESCRIPTION
**Pull request by rdaly525**
Also requires a PR for ast_tools

### CODE REVIEW

**Comment by rdaly525** [comment]
https://github.com/leonardt/ast_tools/pull/22

File: magma/syntax/combinational.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from magma.config import get_debug_mode
 import itertools
 import typing
-from ast_tools.stack import _SKIP_FRAME_DEBUG_STMT
+from ast_tools.stack import _SKIP_FRAME_DEBUG_STMT, SymbolTable
```
Comment (rsetaluri) [inline magma/syntax/combinational.py]: nit: if something is exported from a module (i.e. being imported by someone else) then it shouldn't have a leading `_` :)

Comment (rdaly525) [reply magma/syntax/combinational.py]: I am indifferent, but this is not my code.

**Comment by rdaly525** [comment]
@leonardt, could you republish ast_tools? That is needed to get this passing

**Pull request merged**